src-plugins/manualRegistration/manualRegistration.cpp

```diff
@@ -58,16 +50,11 @@ class manualRegistrationPrivate
 manualRegistration::manualRegistration() : itkProcessRegistration(), d(new manualRegistrationPrivate)
 {
     d->proc = this;
-    d->registrationMethod = NULL ;
-
-    //set transform type for the exportation of the transformation to a file
-    this->setProperty("transformType","Rigid");
```

Comment:
In fact i change it in the next PR: https://github.com/Inria-Asclepios/medInria-public/pull/254
But i'll remove the change here indeed. In this PR this line is not used since the toolbox does not use the unde/redo process (that what i changed in the PR 254)